sw/index.js

```diff
@@ -77,7 +77,7 @@ self.addEventListener('push', async function (event) {
   const { tag } = payload.options
   event.waitUntil((async () => {
     // TIP and EARN notifications simply replace the previous notifications
```

Comment:
you might want to update this comment aswell

Reply:
Thought so as well but then I thought forwarded tips are tips as well, sooo... :smile: